src/components/forms/IconInput.jsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import cx from 'classnames';
+
+import FDS from '../../../lib/dictionary/js/styleConstants';
+
+/**
+ * @param {Object} props react props
+ * @returns {ReactElement}
+ */
+const IconInput = ({ IconLeft, IconRight, children }) => {
+  const classNames = cx('fdsIconInput', {
+    'fdsIconInput--hasStartIcon': Boolean(IconLeft),
+    'fdsIconInput--hasEndIcon': Boolean(IconRight),
```

Comment:
do these need `Boolean` or were you just trying to be more transparent that these values aren't boolean to begin with

Reply:
mostly for clarity - it's technically not necessary